lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -307,12 +293,9 @@ sub ia_base :Chained('base') :PathPart('view') :CaptureArgs(1) {  # /ia/view/cal
             $can_edit = 1;
 
             if ($is_admin) {
-                my $edits = get_edits($c->d, $answer_id);
+                my @edits = get_all_edits($c->d, $c->stash->{ia}->id);
```

Comment:
Why not use `$answer_id` instead of `$c->stash->{ia}->id`? Anyway when I'm logged in as admin and I try to see a single IA Page, and thus this line is executed, I get a DB error. I think it's in the get_all_edits() function...